inc/class-wpseo-replace-vars.php

```diff
@@ -724,7 +724,7 @@ private function retrieve_ct_desc_custom_tax_name( $var ) {
 					$term      = current( $terms );
 					$term_desc = get_term_field( 'description', $term->term_id, $tax );
 					if ( $term_desc !== '' ) {
-						$replacement = trim( strip_tags( $term_desc ) );
+						$replacement = trim( wp_strip_all_tags( $term_desc ) );
```

Comment:
As `wp_strip_all_tags` already trims the content, this does not have to be wrapped again.